### PR TITLE
Li/version check

### PR DESCRIPTION
**Pull request by wanliAlex**
* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
We recommend marqo1 if we detect the marqo version is 1.x.x

* **What is the current behavior?** (You can also link to an open issue here)
we don't have marqo1 recommendation

* **What is the new behavior (if this is a feature change)?**
We recommend marqo1 if we detect the marqo version is 1.x.x

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:
no

### CODE REVIEW

File: src/marqo/index.py [hunk 1]
```diff
@@ -624,10 +623,22 @@ def _marqo_minimum_supported_version_check(self):
...
-                mq_logger.warning(f"Your Marqo Python client requires a minimum Marqo version of "
-                                  f"{minimum_supported_marqo_version()} to function properly, but your Marqo version is {marqo_version}. "
-                                  f"Please upgrade your Marqo instance to avoid potential errors. "
-                                  f"If you have already changed your Marqo instance but still get this warning, please restart your Marqo client Python interpreter.")
+                if versioning_helpers.parse(marqo_version).major == 1:
+                    mq_logger.warning(
+                        f"Your current Marqo Python client requires a minimum Marqo version of "
+                        f"{minimum_supported_marqo_version()} to function properly. "
+                        f"Please upgrade your Marqo instance to avoid potential errors. "
+                        f"If you intended to use Marqo versions 1.x.x, "
```
Comment (farshidz) [inline src/marqo/index.py]: If you intended to use Marqo 1.x, please use Python client 2.x (pip install marqo==2).

File: src/marqo/index.py [hunk 2]
```diff
@@ -624,10 +623,22 @@ def _marqo_minimum_supported_version_check(self):
...
+                        f"please use the 'marqo1' Python client by pip install marqo1. "
+                        f"If you have already changed your Marqo instance but still get this warning, "
+                        f"please restart your Marqo client Python interpreter.")
```
Comment (farshidz) [inline src/marqo/index.py]: please restart your Python interpreter.

File: src/marqo/index.py [hunk 3]
```diff
@@ -624,10 +623,22 @@ def _marqo_minimum_supported_version_check(self):
...
+                else:
+                    mq_logger.warning(f"Your Marqo Python client requires a minimum Marqo version of "
+                                      f"{minimum_supported_marqo_version()} to function properly, "
+                                      f"but your Marqo version is {marqo_version}. "
+                                      f"Please upgrade your Marqo instance to avoid potential errors. "
+                                      f"If you have already changed your Marqo instance but still get this warning, "
+                                      f"please restart your Marqo client Python interpreter.")
```
Comment (farshidz) [inline src/marqo/index.py]: please restart your Python interpreter.

**Review by farshidz — CHANGES_REQUESTED** [review]

File: src/marqo/index.py [hunk 4]
```diff
@@ -624,10 +623,22 @@ def _marqo_minimum_supported_version_check(self):
...
-                                  f"{minimum_supported_marqo_version()} to function properly, but your Marqo version is {marqo_version}. "
-                                  f"Please upgrade your Marqo instance to avoid potential errors. "
-                                  f"If you have already changed your Marqo instance but still get this warning, please restart your Marqo client Python interpreter.")
+                if versioning_helpers.parse(marqo_version).major == 1:
+                    mq_logger.warning(
+                        f"Your current Marqo Python client requires a minimum Marqo version of "
+                        f"{minimum_supported_marqo_version()} to function properly. "
+                        f"Please upgrade your Marqo instance to avoid potential errors. "
+                        f"If you intended to use Marqo versions 1.x.x, "
+                        f"please use the 'marqo1' Python client by pip install marqo1. "
```
Comment (pandu-k) [inline src/marqo/index.py]:
```
please use the 'marqo1' Python client by pip install marqo1. 
```
->
```
please use the 'marqo1' Python client: `pip install marqo1`
```

**Review by farshidz — APPROVED** [review]

**Pull request merged**